src/main/java/com/gudimov/movieland/controller/MovieController.java

```diff
@@ -1,6 +1,7 @@
 package com.gudimov.movieland.controller;
 
 import com.gudimov.movieland.entity.Movie;
+import com.gudimov.movieland.entity.MovieWithGenreCountry;

```

Comment:
Why do we need a separate entity for this?

Reply:
Class MovieWithGenreCountry extends class Movie